packages/web-app-files/src/composables/actions/spaces/useSpaceActionsUploadImage.ts

```diff
@@ -59,13 +59,11 @@ export const useSpaceActionsUploadImage = ({ spaceImageInput }: { spaceImageInpu
       //
       // https://github.com/perry-mitchell/webdav-client/blob/dd8d0dcc319297edc70077abd74b935361bc2412/source/tools/body.ts#L18
       const content = await file.arrayBuffer()
-      const headers = {
+      const headers: Record<string, string> = {
         'Content-Type': 'application/offset+octet-stream'
       }
 
-      if (file.lastModifiedDate) {
-        headers['X-OC-Mtime'] = '' + file.lastModifiedDate.getTime() / 1000
```

Comment:
This seems to be removed from the API: https://developer.mozilla.org/en-US/docs/Web/API/File/lastModifiedDate